radix-engine/src/kernel/call_frame.rs

```diff
@@ -224,8 +224,8 @@ pub enum StableReferenceType {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub struct TransientReference {
-    ref_count: usize,
-    ref_origin: ReferenceOrigin,
+    pub ref_count: usize,
```

Comment:
Are these `pub` still required?

Reply:
Yup, no longer needed. I've reverted the changes.